multi-cluster-spring-boot/image-mirror-example/.applier/params/build-dev

```diff
@@ -0,0 +1,9 @@
+APPLICATION_NAME=spring-rest
+NAMESPACE=multicluster-spring-boot-dev
+PIPELINE_REPOSITORY_URL=https://github.com/redhat-cop/container-pipelines.git
+PIPELINE_REPOSITORY_REF=multi-cluster-image-mirror
+PIPELINE_REPOSITORY_CONTEXT_DIR=multi-cluster-spring-boot/image-mirror-example
+SRC_API_URL=<SRC_API_URL>
```

Comment:
Maybe we should add documentation on this value needs to be set before running applier?